test/development/slow-module-detection/index.test.ts

```diff
@@ -0,0 +1,83 @@
+import { createNext, NextInstance } from 'e2e-utils'
+import { fetchViaHTTP } from 'next-test-utils'
+;(process.env.TURBOPACK ? describe.skip : describe)(
+  'Slow Module Detection',
+  () => {
+    let next: NextInstance
+
+    beforeAll(async () => {
+      next = await createNext({
+        files: {
```

Comment:
We should hardcode files whenever possible so that local testing is easier (`pnpm next dev test/...`). The only file that doesn't fit this criteria is the auto-generated `utils/slow-module.js`. I'd just put that into git-ignore so that people can still easily tinker with its size.